### PR TITLE
PRESS2-162 Add tests for Tax Info Step

### PR DESCRIPTION
**Pull request by arunshenoy99**

### CODE REVIEW

**Review by officiallygod — APPROVED** [review]

File: tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/ecommerce/tax-info.cy.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    // it('Is Accessible', () => {
+	// 	cy.wait(500);
+	// 	cy.checkA11y();
+	// });
+
+    it('Checks if Drawer opened', () => {
+        cy.get('.nfd-onboarding-drawer__panel-inner')
+            .scrollIntoView()
+            .should('be.visible');
+    });
```
Comment (Yashita101) [inline tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/ecommerce/tax-info.cy.js:22]: Can we add a testcase to check if the respective page is selected and highlighted in the `Drawer Panel` ? As I remember David mentioning that we should check that too
image

Comment (arunshenoy99) [reply tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/ecommerce/tax-info.cy.js:22]: Line 24-29 is what you missed looking at xD

Comment (arunshenoy99) [reply tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/ecommerce/tax-info.cy.js:22]: Although there is a typo I'll fix that in the test case name

Comment (arunshenoy99) [reply tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/ecommerce/tax-info.cy.js:22]: Fixed typo

Comment (Yashita101) [reply tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/ecommerce/tax-info.cy.js:22]: great, thanks! Rest all looks good

**Review by Yashita101 — APPROVED** [review]

**Pull request merged**